README.md

```diff
@@ -110,7 +110,21 @@ with more than one breaking release of a package, add multiple entries such as
 StaticArrays = "0.10, 0.11"
 ```
 
-See [the `Pkg`
+This attests that the package is compatible with both versions of `StaticArrays`
+even though, according to [semantic versioning](https://semver.org/), the `0.11`
```

Comment:
Should we link to the docs explaining julia extension of SemVer instead?

Reply:
Sure, that makes sense. Do you think this would be the right section? https://julialang.github.io/Pkg.jl/stable/compatibility/#Version-specifier-format-1

Reply:
We need to link to a newer version, so it includes this section abojut leading zeros
https://julialang.github.io/Pkg.jl/dev/compatibility/#Behavior-of-versions-with-leading-zeros-(0.0.x-and-0.x.y)-1

Reply:
Ok, done